spinta/datasets/commands/check.py

```diff
@@ -1,14 +1,45 @@
 from spinta import commands
 from spinta.components import Context
 from spinta.datasets.components import Dataset, Resource
+import re
 
 
 @commands.check.register()
 def check(context: Context, dataset: Dataset):
     for resource in dataset.resources.values():
         commands.check(context, resource)
 
+    rc = context.get('rc')
+
+    if rc.get('names'):
+        name = dataset.name
+        is_lowercase = len([i for i in name if i.islower()] + re.findall('/', name)) == len(name)
+
+        # Dataset name check
+        if not is_lowercase or name.startswith('/') or name.endswith('/') or re.findall('//', name):
+            raise Exception(f'Dataset name {name} is not correct.')
```

Comment:
Create a custom class in `spinta/exceptions.py` called `InvalidName`, which should extend `UserError`.